### ISSUE TITLE
webmusic.live

### ISSUE THREAD

**Issue by ghajini** [uBlockOrigin/uAssets#7900]
### URL(s) where the issue occurs

`http://webmusic.live/music/mobile/hindi/file.php?id=43167`

### Describe the issue

popup ads

### Versions

- Browser/version: firefox 80.0.1
- uBlock Origin version: 1.29.2

### Settings

uBO's default settings
-

**Comment by stephenhawk8054** [comment]
In this site: `https://reyohoho.github.io/reyohoho/#61249`, this connection 

```yml
https://aj2635.bid/zzjvcxiIpdeoOECJJu8G8uepAT-OAwdErQ4NLZg85splTGCYAWBtHsbpPDXtNubow6HYGFaVt5MsEthhAX64r8eF1fzQiufXj2aqtFkRU65MKMXnjgz4?vpaid=false&ch=reyohoho.github.io
```

is not blocked in uBOL. I think maybe because `/^https:\/\/(?:cdn77\.)?aj\d{4}\.bid\/[-_0-9A-Za-z]{80,}\?/$xhr,3p,to=bid` is discarded by DNR. Anybody has a good filter for uBOL, or should we block it per-site?

**Comment by mapx-** [comment]
I see no such (rejected) regex in the log.txt
https://github.com/uBlockOrigin/uBOL-home/blob/main/chromium/log.txt

**Comment by stephenhawk8054** [comment]
> rule works fine in AG mv3

I don't think so. When I added it as custom filter in AG MV3, the XHR connection still loads for me:

![Image](https://github.com/user-attachments/assets/48db53aa-08e3-46c9-a93c-9f1b12ee8c51)

Note that when you see AG log puts a red (blocked) line, it just means that the connection is supposed to be blocked by the regex, but the real blocking depends on browsers' DNR

![Image](https://github.com/user-attachments/assets/6f1e3222-8e8e-4225-8729-a39a95b2c269)

https://adguard.com/kb/adguard-browser-extension/solving-problems/debug-rules/

> In [AdGuard for Chrome MV3](https://adguard.com/kb/adguard-browser-extension/mv3-version/), the Filtering log only shows the approximate rules that were applied, which we call “assumed rules”. This is because the browser doesn’t provide details about which specific declarative rules were used unless the extension is in an “unpacked” format. To get precise information, you’ll need to install the unpacked version of the extension in your browser yourself.

**Comment by stephenhawk8054** [comment]
> I see no such (rejected) regex in the log.txt https://github.com/uBlockOrigin/uBOL-home/blob/main/chromium/log.txt

Yeah, the log is just for the filters that are rejected by DNR pre-compiled. There are regex cases where you can only know if DNR rejects the regex when you loads the extension at run-time (in the logger as gorhill said): https://github.com/uBlockOrigin/uAssets/issues/26671#issuecomment-2566496560

**Comment by mapx-** [comment]
```adb
/^https:\/\/(?:cdn77\.)?aj\d{4}\.bid\/.*\?/$xhr,3p,to=bid
```

**Comment by stephenhawk8054** [comment]
Yeah, I think that could work.

**Comment by Yuki2718** [comment]
I don't see the request, a connectioon to `4f463c79.obrut.show` is rejected. But why
`/^https:\/\/(?:cdn77\.)?aj\d{4}\.bid\/.*\?/$xhr,3p,to=bid` works if `/^https:\/\/(?:cdn77\.)?aj\d{4}\.bid\/[-_0-9A-Za-z]{80,}\?/$xhr,3p,to=bid` doesn't?

**Comment by mapx-** [comment]
aj* crap only if you click play

**Comment by mapx-** [comment]
`[\w\W]{80,}` does not work too

**Comment by stephenhawk8054** [comment]
> But why
`/^https:\/\/(?:cdn77\.)?aj\d{4}\.bid\/.*\?/$xhr,3p,to=bid` works if `/^https:\/\/(?:cdn77\.)?aj\d{4}\.bid\/[-_0-9A-Za-z]{80,}\?/$xhr,3p,to=bid` doesn't

I think it's because of `{80,}`. The `isRegexSupported()` is purely based on browsers deciding about "performance" themselves, so maybe they think that regex is too expensive?

**Comment by mapx-** [comment]
this is working

```adb
/^https:\/\/(?:cdn77\.)?aj\d{4}\.bid\/[^?]{30,}\?/$xhr,3p,to=bid
```

when >30 does not work

**Comment by mapx-** [comment]
```
{n,m} quantifier
Chrome: supports, but many rules fail with memoryLimitExceeded
```

https://github.com/w3c/webextensions/issues/344

**Comment by Yuki2718** [comment]
Looks now the site was parked.

**Comment by stephenhawk8054** [comment]
Which site?

**Comment by Yuki2718** [comment]
> Which site?

`webmusic.live`
